app/trip_cost.py

```diff
@@ -0,0 +1,17 @@
+from app.customer import Customer
+from app.shop import Shop
+
+
+def calculate_trip_cost(
+        customer: Customer,
+        shop: Shop,
+        fuel_price: float
+) -> float:
+    distance = (abs(customer.location[0] - shop.location[0])
+                ** 2 + abs(customer.location[1] - shop.location[1])
+                ** 2) ** 0.5
+    cost = ((distance * 2 * customer.car_consumption / 100 * fuel_price)

```

Comment:
Ensure that `customer.car_consumption` is a valid number. If it's not set or is invalid, this calculation will raise an error.